plugins/source/azure/resources/services/subscriptions/locations.go

```diff
@@ -53,13 +55,25 @@ func Locations() *schema.Table {
 				Type:     schema.TypeString,
 				Resolver: schema.PathResolver("Type"),
 			},
+			{
+				Name:     "subscription_id",
+				Type:     schema.TypeUUID,
+				Resolver: resolveSubscriptionID,
+			},
 		},
 	}
 }
 
-func fetchSubscriptionsLocations(ctx context.Context, meta schema.ClientMeta, _ *schema.Resource, res chan<- interface{}) error {
+func resolveSubscriptionID(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
```

Comment:
Not sure why, but getting the subscription ID directly from the location sets a `nil` value in the database.
I'll look into it and probably revert this custom resolver

Reply:
Done